src/components/Icon/icons/Phone.tsx

```diff
@@ -0,0 +1,9 @@
+import * as React from 'react';
+
+export default function Phone(props: any) {
```

Comment:
is this an example Icon to copy+paste from?

Reply:
I cannibalized what came out of the svgtoreact cli.

Reply:
We would make a proper template, and ideally create our own command line tool if we go in this direction.